common/lib/dependabot/pull_request_creator/codecommit.rb

```diff
@@ -197,12 +197,11 @@ def pull_requests_for_branch
 
       sig { void }
       def create_commit
-        author = author_details&.slice(:name, :email, :date)
-        author = nil unless author&.any?
+        author = author_details&.slice(:name, :email, :date)&.values&.first
 
         codecommit_client_for_source.create_commit(
           branch_name,
-          author,
+          author.to_s,
```

Comment:
We shouldn't call `to_s` on this, since we made the param `nillable`, otherwise we'd end up passing an empty string when the value is `nil` @thavaahariharangit